pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/loader/SegmentPreProcessor.java

```diff
@@ -39,196 +37,177 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
 /**
  * mmap()'s the segment and performs any pre-processing to generate inverted index
  * This can be slow
  */
 public class SegmentPreProcessor implements AutoCloseable {
-  private static Logger LOGGER = LoggerFactory.getLogger(SegmentPreProcessor.class);
+  private final static Logger LOGGER = LoggerFactory.getLogger(SegmentPreProcessor.class);
 
-  private final File indexDir;
-  private SegmentDirectory segmentDirectory;
-  private final SegmentMetadataImpl metadata;
-  private final Optional<IndexLoadingConfigMetadata> indexConfig;
-  //private IndexLoadingConfigMetadata indexConfig;
+  private final File _indexDir;
```

Comment:
Don't prefix these with _ (Code is currently hodgepodge but we agreed to follow google coding convention)